### DISCUSSION TITLE
Limitation on sending Statsd metrics

### DISCUSSION THREAD

**Discussion by amir5142** [netdata/netdata#16921] [Q&A]
Is there any limitation on sending metrics to netdata using StatsD? I'm experiencing difficulty sending a large volume of StatsD-formatted data to netdata. Are there any restrictions or considerations I should be aware of? my netdata version is: v1.28.0-196-nightly and i try something like this for test, but i can't get result more than 260 metrics in netdata:
for 
for i in {1..500}
do
   echo "metric$i:1000|g"|nc -w 1 -u 127.0.0.1  8125
done

**Comment by ilyam8** [comment]
@amir5142, latest stable version is [v1.44.1](https://github.com/netdata/netdata/releases/tag/v1.44.1).
